src/components/Events/AdminEvents/adminAddEvent.js

```diff
@@ -111,7 +111,17 @@ export default class AdminAddEvent extends React.Component {
 						<div className="input-row">
 							<div className="input-field">
 								<p>Committee</p>
-								<input type="text" value={this.state.event.committee} name="committee" onChange={this.handleChange} />
+								<select value={this.state.event.committee} name="committee" onChange={this.handleChange}>
+									<option value="AI">AI</option>
```

Comment:
For the options in the dropdown, can you dynamically set them by reading in values from src/config.js?

Reply:
Oh that's what you meant. Got cha!!